templates/lynx/meta.yaml

```diff
@@ -0,0 +1,41 @@
+name: Lynx
+description: A fullstack application using the MEVN stack to shorten your URLs.
+instructions: null
```

Comment:
We should maybe add a instructions like "Registration is disabled by default, but the first registration will always be allowed."